easybuild/easyconfigs/n/ngs-utils/ngs-utils-22.01.1.eb

```diff
@@ -21,15 +21,15 @@ dependencies = [
 ]
 
 postinstallcmds = [
-    'chmod -R 775 %(installdir)s/bin/*.sh',
-    'chmod -R 775 %(installdir)s/bin/*.bash',
-    'chmod -R 775 %(installdir)s/bin/*.pl',
-    'chmod -R 775 %(installdir)s/bin/*.py',
-    'chmod -R g+rwX,o+rX,o-w %(installdir)s/bin/*'
+    'chmod -R 775 %(installdir)s/*.sh',
+    'chmod -R 775 %(installdir)s/*.bash',
+    'chmod -R 775 %(installdir)s/*.pl',
+    'chmod -R 775 %(installdir)s/*.py',
+    'chmod -R g+rwX,o+rX,o-w %(installdir)s/*'
```

Comment:
Waarom deze wijziging? De scripts zitten voor zover ik kan zien nog altijd in een bin subdir, wat ook wel zo handig is, want die wordt dan automatisch toegevoegd aan ```${PATH}```, waardoor ze gevonden worden zonder expliciet absoluut pad.

Reply:
dat ging niet goed, het is weer terug gezet.